lib/resources/forms.js

```diff
@@ -161,7 +161,7 @@ module.exports = (service, endpoint) => {
       .then(([form, project]) =>
         Promise.all([
           auth.canOrReject('form.update', form),
-          auth.canOrReject('dataset.list', project)
+          auth.canOrReject('dataset.read', project)
```

Comment:
Right now the diff will only include a single dataset, right? In the future, might it include multiple datasets? If so, `dataset.list` actually feels more right to me than `dataset.read`. I think of `dataset.read` as being about an individual dataset.

Reply:
I wasn't totally sure what to put here. Good point that there could be multiple datasets!